File: Latihan B/PHP/Disk.php
<?php
class Disk {
    private $type;
    private $capacity;
    private $price;

    // konstruktor
    function __construct($type, $capacity, $price) {
        $this->type = $type;
        $this->capacity = $capacity;
        $this->price = $price;
    }

    // set untuk semua variable
    function setType($type) {
        $this->type = $type;
    }
    function setCapacity($capacity) {
        $this->capacity = $capacity;
    }
    function setPrice($price) {
        $this->price = $price;
    }

    // get untuk semua variable
    function getType() {
        return $this->type;
    }
    function getCapacity() {
        return $this->capacity;
    }
    function getPrice() {
        return $this->price;
    }

    // tampil untuk disk
    function printDisk() {
        echo "Tipe Disk : ".  $this->getType(). "<br/>";
        echo "Kapasitas Disk : ". $this->getCapacity(). "<br/>";
        echo "Harga Disk : ". $this->getPrice(). "<br/>";
    }

    // destruktor
    function __destruct() {}
}
?>

File: Latihan B/JAVA/Disk.java
public class Disk {

    private String type;
    private int capacity;
    private int price;

    // konsturktor
    public Disk(){

    }
    public Disk(String type, int capacity, int price) {
        this.type = type;
        this.capacity = capacity;
        this.price = price;
    }

    //set semua variable
    public void setType(String type) {
        this.type = type;
    }
    public void setCapacity(int capacity) {
        this.capacity = capacity;
    }
    public void setPrice(int price) {
        this.price = price;
    }

    // get semua variable
    public String getType() {
        return type;
    }
    public int getCapacity() {
        return capacity;
    }

    public int getPrice() {
        return price;
    }

    // tampil untuk Disk
    public void printDisk() {
        System.out.println("  Tipe Disk          : " + type);
        System.out.println("  Kapasitas Disk     : " + capacity);
        System.out.println("  Harga Disk         : " + price);
    }
}